specification/ws.extensions/x-kaazing-ping-pong/src/main/scripts/org/kaazing/specification/ws.extensions/x-kaazing-ping-pong/wsn/client.should.reply.to.extended.ping.with.extended.pong/request.rpt

```diff
@@ -0,0 +1,41 @@
+connect tcp://127.0.0.1:8001
+connected
+
+# Connection request
+write "GET /echo?.kl=Y HTTP/1.1\r\n"
+write "Upgrade: websocket\r\n"
+write "Connection: Upgrade\r\n"
+write "Host: localhost:8001\r\n"
+write "Origin: http://localhost:8001\r\n"
+write "Sec-WebSocket-Key: nDaimG37f4nUqogPypithw==\r\n"
+write "Sec-WebSocket-Version: 13\r\n"
+write "Sec-WebSocket-Extensions: x-kaazing-ping-pong\r\n"
+write "\r\n"
+
+# 101 Response
+read "HTTP/1.1 101 Web Socket Protocol Handshake\r\n"
+read "Connection: Upgrade\r\n"
+read "Date:" /.*/ "\r\n"
+read "Sec-WebSocket-Accept:" /.*/ "\r\n"
+# The 4 control bytes for ping-pong extension are 0x01010102:
+read "Sec-WebSocket-Extensions: x-kaazing-ping-pong; 01010102"
+read "\r\n"
+read "Server: Kaazing Gateway\r\n"
+read "Upgrade: websocket\r\n"
+read "\r\n"
+
+# binary-encoded balancer directive - NO balance needed
+read [0x82 0x04 0xEF 0x83 0xBF 0x4E]
+
+# Read extended PING
+read [0x81 0x06 0x01 0x01 0x01 0x02 0x09 0x00]
+
+# Send extended masked PONG response (unmasked form is [0x81 0x06 0x01 0x01 0x01 0x02 0x0a 0x00])
+#write [0x81 0x86 0x01 0x01 0x01 0x01 0x00 0x00 0x00 0x03 0x0b 0x01]
```

Comment:
This should not be commented out. When you move to use http k3po transport instead of tcp it will become much easier to do the masking!